boards/nxp/imx93_evk/board.cmake

```diff
@@ -1 +1,8 @@
+#
+# Copyright 2025 NXP
+#
 # SPDX-License-Identifier: Apache-2.0
+
+board_runner_args(jlink "--device=MIMX9352_A55_0" "--no-reset" "--flash-script=${CMAKE_BINARY_DIR}/zephyr/runner.jlinkscript")
```

Comment:
does the m33 core supports jlink as well? as it stands, nothing will prevent the user to try to "west flash" if they build for the m33 target. You may want to put this into an "if(CONFIG_SOC_....)" block if it's really meant to be restricted to M55 core.

Reply:
Good catch, have changed to only feasible for A55 Core, thanks.